core/services/ocr2/delegate.go

```diff
@@ -631,6 +636,26 @@ func (d *Delegate) ServicesForSpec(jb job.Job) ([]job.ServiceCtx, error) {
 			return nil, errors.Wrap(err2, "ocr2keepers plugin config validation failure")
 		}
 
+		enc := v1.NewEncoder()
+
+		wrapper := &logWriter{l: ocrLogger}
+		prefixedLogger := stdlog.New(wrapper, "[keepers-plugin] ", stdlog.Lshortfile)
+
+		pollingObserver := polling.NewPollingObserver(
+			prefixedLogger,
+			rgstry,
+			polling.NewKeyProvider(rgstry),
+			10,
+			10,
+			20*time.Second,
+			time.Minute,
+			time.Minute,
+			coordinator.NewReportCoordinator(rgstry, time.Minute, time.Minute, logProvider, 5, prefixedLogger),
```

Comment:
@EasterTheBunny Where should the values for these fields come from? The config that exposes cache expiration/eviction values? Or somewhere else?

Reply:
Oh actually nevermind, I see where they're meant to come from now! And just one needs to come from the offchain config which is easy to handle! 👍

Reply:
@EasterTheBunny based on how the offchain config only becomes available after this point, we have to configure the coordinator via the observer, is that okay with you?

Reply:
Works for me